client/daemon/peer/peertask_file_test.go

```diff
@@ -103,7 +103,7 @@ func TestFilePeerTask_BackSource_WithContentLength(t *testing.T) {
 			Url:      "http://localhost/test/data",
 			Filter:   "",
 			BizId:    "d7y-test",
-			UrlMata:  nil,
+			UrlMeta:  nil,
```

Comment:
![image](https://user-images.githubusercontent.com/16535906/122325853-e375fa00-cf5d-11eb-9b48-eec5dddf7d26.png)

Reply:
No initialization is required